Makefile

```diff
@@ -50,7 +50,7 @@ check_types:
 		-o -path ./.eggs \
 		-o -path ./gen/python/setup.py \
 		-o -path ./test/symforce_function_codegen_test_data \
-		\) -prune -o -name "*.py" -print) \
+		\) -prune -o -name "*.py" -not -path "./gen/python/build/*" -print) \
```

Comment:
My hatred for these `find` commands grows every time we modify them.  Can we just add a line `-o -path ./gen/python/build \` above instead of adding this here?  Or is there some reason this one is different?

Reply:
We can indeed. Made the change.